components/BaseComponents.js

```diff
@@ -0,0 +1,108 @@
+import styled from 'styled-components/native';
+
+export const TextButton = styled.Text`
+  font-family: poppins-regular;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #008550;
+`;
+
+export const TextButtonPressed = styled(TextButton)`
+  color: #004e14;
```

Comment:
Do these color's have a corresponding names in Figma? And do these colors already exist in a globals styles variable file that we could import and use? 

For readability and reusability would be great if the colors would read something with a name

`export const TextButtonPressed = styled(TextButton)`
  color: ${primary-green};
`;
`

Reply:
Switching to camel case for the naming because that seems to be the norm!